src/qt/bitcoingui.cpp

```diff
@@ -461,7 +461,7 @@ void BitcoinGUI::createTrayIcon()
     trayIconMenu->addAction(quitAction);
 #endif
 
-    notificator = new Notificator(tr("p-qt"), trayIcon);
+    notificator = new Notificator(qApp->applicationName(), trayIcon);
```

Comment:
It was updated to use a dynamic solution in favor of a static string such as "p-qt" (also this name is a bit silly you would expect something more like paycoin-qt which is a bit more friendly).

It will now get this name using qApp->applicationName() from ![bitcoin.cpp#L236](https://github.com/PaycoinFoundation/paycoin/blob/master/src/qt/bitcoin.cpp#L236).

It saves updating the name 'Paycoin-Qt' in multiple locations and there's no need to translate it.